trtools/utils/tr_harmonizer.py

```diff
@@ -36,18 +35,18 @@ def __repr__(self):
         return '<{}.{}>'.format(self.__class__.__name__, self.name)
 
 
-def GetVCFType(vcfreader, vcftype: Union[str, VcfTypes] = "auto"):
+def GetVCFType(vcfreader: cyvcf2.VCF, vcftype: Union[str, VcfTypes] = "auto"):
     """Infer the vcf type of the reader.
 
-    If vcftype is "auto", try to infer types of reader.
-    If it is not auto, check to make sure vcftype is correct and return that string.
+    If vcftype is not auto just return that string (assuming it corresponds
+    to a valid vcf type). If it auto, try to infer the vcftype from vcfreader.
```

Comment:
Good catch. This seems to be a similar method as InferVCFType but with a less clear specification of how it handles errors, so I'm just deleting it. The 2 places which use this method instead of InferVCFType will be updated to use that in their respective PRs.